torch/fx/proxy.py

```diff
@@ -104,6 +104,61 @@ def __init__(self, graph: Graph):
 class TraceError(ValueError):
     pass
 
+
+def _create_friendly_names(args: Tuple[Any, ...], kwargs: Dict[str, Any], frames_up : int):
```

Comment:
frames_up: seems fragile, this will break all the time. Right now it manifests as not encapsulating `_assign_friendly_name` but it will have later effects on code clarity since this has to be threaded through everything. Maybe instead look for the upper-most frame defined in the proxy.py file.